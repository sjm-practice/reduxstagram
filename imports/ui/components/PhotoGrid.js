// @flow
import React, {
  Component,
} from "react";

// eslint-disable-next-line react/prefer-stateless-function
class PhotoGrid extends Component {
  render() {
    return (
      <div className="photo-grid">
        photo grid
      </div>
    );
  }
}

export default PhotoGrid;
