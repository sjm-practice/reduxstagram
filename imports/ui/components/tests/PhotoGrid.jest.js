// @flow
/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */

import React from "react";
import renderer from "react-test-renderer";
import PhotoGrid from "../PhotoGrid";

describe("<PhotoGrid />", function () {
  it("matches render snapshot", function () {
    const tree = renderer.create(<PhotoGrid />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

