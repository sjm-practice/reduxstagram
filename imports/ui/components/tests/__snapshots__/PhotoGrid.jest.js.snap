exports[`<PhotoGrid /> matches render snapshot 1`] = `
<div
  className="photo-grid">
  photo grid
</div>
`;
